docs/contributor/image_mode.md

```diff
@@ -63,8 +63,19 @@ RUN firewall-offline-cmd --zone=public --add-port=80/tcp && \
     firewall-offline-cmd --zone=public --add-port=30000-32767/tcp && \
     firewall-offline-cmd --zone=public --add-port=30000-32767/udp
 
-# Recursively make the root filesystem subtree shared as required by OVN images
-CMD ["/bin/sh", "-c", "mount --make-rshared / ; exec /sbin/init"]
+# Create a systemd unit to recursively make the root filesystem subtree
+# shared as required by OVN images
+RUN cat > /etc/systemd/system/microshift-make-rshared.service <<'EOF'
+[Unit]
+Description=Make root filesystem shared
+Before=microshift.service
```

Comment:
I forgot to mention this in my previous comment but we should have `ConditionVirtualization=container` here; don't want to affect the main booted path!

Reply:
![image](https://github.com/user-attachments/assets/3a62d5da-457d-4fae-ad01-3833219023de)

Reply:
Hmm this should work but I admit to not testing it locally, did you get it to work or not?

Reply:
Yes, it's working OK according to my tests. 
I provided the service status above that comes from a VM. For a container, the service runs successfully.
All good, IMO.